middlewares/authorizeOwner.js

```diff
@@ -0,0 +1,35 @@
+const usersController = require('../controllers/usersController')
+/**
+ * Middleware to validate the authorized routes to fetch open PRs
```

Comment:
I think this should be something like this
`Middleware to validate the authorized routes to be able to create & Update tasks`